### DISCUSSION TITLE
How to get rid of tiny gap infill lines

### DISCUSSION THREAD

**Discussion by evolutionlures** [OrcaSlicer/OrcaSlicer#5466] [Q&A]
While printing in TPU I have noticed the printer will try to add gap infill to tiny areas of my print. Since it is tpu and transparent. These little marks inside the wall show up and an unclean line in the final print. I have checked every setting I can find and I cannot get it to stop trying to fill in these tiny areas with dashed lines. Anyone have any luck getting the slicer to not do this?

<img width="946" alt="Screenshot 2024-05-26 at 8 50 30 AM" src="https://github.com/SoftFever/OrcaSlicer/assets/170131087/e8c19a44-46bd-4d6a-8e21-4035808f1836">

**Comment by vgdh** [comment]
I don't think full transparent is achievable.
But you can enable  **fill gaps** in **Strength section**

**Comment by evolutionlures** [comment]
I'm not trying to fill the gaps actually, since tpu spreads out and is very sticky already, just not having the dashed lines is what I need. The less moves you do with tpu, the better it prints. I don't care if it's super transparent really. It's more about flow consistency.  This is something cura does by default, so the prints actually come out cleaner in cura. Sent from my Galaxy
-------- Original message --------From: vgdh ***@***.***> Date: 5/26/24  10:09  (GMT-06:00) To: SoftFever/OrcaSlicer ***@***.***> Cc: evolutionlures ***@***.***>, Author ***@***.***> Subject: Re: [SoftFever/OrcaSlicer] How to get rid of tiny gap infill lines (Discussion #5466) 
I don't think full transparent is achievable.
But you can enable  fill gaps in Strength section
